data/110/894/262/7/1108942627.geojson

```diff
@@ -4,6 +4,8 @@
   "properties": {
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
+    "edtf:deprecated":"12-21-2020",
```

Comment:
Agree with this one.. looks like a hill / location of Parliament: https://en.wikipedia.org/wiki/Dealul_Spirii